api/drivers/aws/driver.js

```diff
@@ -312,6 +312,67 @@ class AWSDriver extends Driver {
   }
 
   /**
+   * Start the specified machine.
+   *
+   * @method startMachine
+   * @return {Promise[Object]}
+   */
+  startMachine(machine) {
+    return new Promise((resolve, reject) => {
+      var params = {
+        InstanceIds: [
+          machine.id
+        ],
+        DryRun: false
+      };
+      this._client.ec2.startInstances(params, (err) => {
+        if (err) {
+          return reject(err);
+        } else {
+          params = {
+            DryRun: false,
+            InstanceIds: [
+              machine.id
+            ]
+          };
+          this._client.ec2.describeInstances(params, (err, data) => {
```

Comment:
This is the role of the poll of `refresh` in `MachineService` to actualize such information.